packages/logging-interceptor/src/logging.interceptor.ts

```diff
@@ -149,7 +170,9 @@ export class LoggingInterceptor implements NestInterceptor {
     const options: LogOptions | undefined = Reflect.getMetadata(METHOD_LOG_METADATA, context.getHandler());
 
     // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
-    const maskedBody = options?.mask?.request ? this.maskData(body, options.mask.request) : body;
+    let maskedBody = options?.mask?.request ? this.maskData(body, options.mask.request) : body;
+    maskedBody = this.truncate(maskedBody, options);
```

Comment:
Not a big fan of defining a `let` if it is just overridden once right after. Defining a let require an extra effort of reviewing to see where it is set in all the following lines.

```suggestion
    const maskedBody = options?.mask?.request ? this.maskData(body, options.mask.request) : body;
    const truncatedMaskedBody = this.truncate(maskedBody, options);
```

Reply:
Fixed in https://github.com/algoan/nestjs-components/pull/935/commits/aa74da3e29e10c2655e37972e430e52610d6f229